enclave-tls/src/sgx/Makefile

```diff
@@ -23,7 +23,7 @@ Enclave_C_Files := \
   ecalls.c ../enclave_quotes/sgx-ecdsa/ecalls.c \
   ../tls_wrappers/wolfssl-sgx/ecalls.c \
   ../crypto_wrappers/wolfcrypt-sgx/ecalls.c
-Enclave_Cflags += -DSGX_ENCLAVE
+Enclave_Cflags += -DSGX_ENCLAVE -DWOLFSSL_SGX_WRAPPER
```

Comment:
wolfssl has defined "WOLFSSL_SGX" for trusted codes and it is defined in . Is it not enough?

I see you are trying to use WOLFSSL_SGX_WRAPPER as an identify of trusted enclave codes.

Reply:
`WOLFSSL_SGX`  can use to replace `SGX_ENCLAVE` which is for trusted codes.
`WOLFSSL_SGX_WRAPPER` is used to distinguish different code implementations of `wolfssl` and `wolfssl-sgx`(untrusted code).
For example, the code in `un_negotiate.c` is included in `wolfssl` and `wolfssl-sgx` (untrusted code, ocalls.c) with different implemetation.